README.md

```diff
@@ -0,0 +1,91 @@
+# dss-bridge
+
+Connect an instance of MCD to a cross-chain guest instance. The `DomainHost` is designed to plug into the Host's dss instance and the `DomainGuest` controls the Guest dss instance on the remote chain. For example, `DomainHost` could be plugged into the MCD master instance on Ethereum and a `DomainGuest` controller would be plugged into the Optimism slave instance of MCD. This is recursive, so that when layer 3s come online we can use another Host/Guest pairing on the L2.
+
+![dss-bridge](https://imgur.com/uEruNWB.png)
+
+Each contract is abstract as it requires a chain-specific messaging service, so one of these needs to be extended for each unique message interface.
+
+We assume any messaging bridge will guarantee messages will be executed exactly once, but does not enforce the ordering of messages. Messaging ordering for sensitive operations is enforced at the application layer (`DomainHost` or `DomainGuest`). If the messaging bridge does not protect against censorship then we cannot provide gaurantees that the bridge will function correctly.
+
+## Supported Operations
+
+### `DomainHost.deposit(address to, uint256 amount)`
+
+Standard deposit mechanism. Locks local DAI and mints canonical DAI on the guest domain.
+
+This will trigger a call to `DomainGuest.deposit(address to, uint256 amount)`.
+
+### `DomainGuest.withdraw(address to, uint256 amount)`
+
```

Comment:
Re: content that was here before
```suggestion
### `DomainGuest.release()`
```



We removed this in this file as it isn't here in changes but is in v1.

However, in v1 it is permissionless (per this description) and makes an internal call to DomainHost, however now we have release on DomainHost which is external (it was internal before) and is authed (not permissionless).

Given that this was here before, do we want to keep it in README as a supported operation? In my view this hasn't been 'deleted', it has just shifted from DomainGuest to DomainHost and has become auth instead of permissionless.

Up to you as to what we are elaborating in this readme.

Reply:
Yes true, we need to add it back to the readme as `DomainHost.release`.

Reply:
Added.